server/planning/events/events_template.py

```diff
@@ -179,6 +180,7 @@ def _fill_event_template(self, doc):
             "event_contact_info",
             "subject",
             "occur_status",
+            "language",
```

Comment:
Similar here, it would be better IMO to not specifically provide what fields to copy, but rather what fields to exclude. We have something in place already for Event Duplicate action:
https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/events/events_duplicate.py#L63

Might be good to move the declaration for the list of fields to remove into a common file as well.